docs/tutorials/integrations/nginx.md

```diff
@@ -0,0 +1,374 @@
+---
+title: Tutorial - NGINX Proxy
+layout: page
+section: tutorials
+description: Conjur Tutorial - NGINX
+---
+
+If you're deploying Conjur in production, you need to set up Transport Layer
+Security so that requests made to the server are encrypted in transit.
+
+*Note: if you use Conjur Enterprise, we handle this for you using an audited
+implementation that works similar to the technique described here.*
+
+## First: A Brief Primer On Transport Layer Security (TLS)
+
+This gets at the heart of the issue we're trying to address with this tutorial.
+
+Suppose you install Conjur and make it available at `http://conjur.local`. Then
+you configure clients to fetch secrets from that address. When they
+[authenticate][authn] using the API, they provide their identity by sending
+their API key to the Conjur server. The Conjur server validates that the
+identity is authentic, then checks that the provided ID is authorized to
+[retrieve the secret][secret-get]. Assuming this check passes, the Conjur server
+returns the secret value to the client.
+
+However, this flow would be vulnerable in two ways. Suppose I were to
+impersonate the Conjur server and listen with my own illegitimate server on
+`http://conjur.local`. Then when your client goes to fetch a secret, I can take
+the API key you send and impersonate you to the real Conjur server. Now I
+control your identity and can learn your secrets without you finding out. This
+is called a **man in the middle attack**.
+
+Even if I'm not able to impersonate the Conjur server, I could still learn your
+secrets by joining your network and listening for traffic coming and going from
+the Conjur server. This is called **passive surveillance**.
+
+### TLS defeats passive and active attacks
+
+Transport Layer Security allows your client to verify that it's talking to the
+real Conjur server, and it uses standard secure technology to encrypt your
+secrets in transit. This means:
+
+* Your Conjur server will be `https:` instead of `http:`, just like a secure
```

Comment:
Maybe "Your Conjur URL will be prefixed by `https` instead of `http`" instead?

Reply:
changed